api-tools/src/test/scala/com/lightbend/lagom/internal/api/tools/ServiceDetectorSpec.scala

```diff
@@ -36,7 +37,36 @@ class ServiceDetectorSpec extends WordSpec with Matchers with Inside {
           |]
         """.stripMargin
 
-      val actualJsonString = ServiceDetector.services(this.getClass.getClassLoader)
+      val javaServiceDiscovery = "com.lightbend.lagom.internal.javadsl.server.JavadslServiceDiscovery"
+      val actualJsonString = ServiceDetector.services(this.getClass.getClassLoader, javaServiceDiscovery)
+      Json.parse(actualJsonString) shouldBe Json.parse(expectedJsonString)
+    }
+
+    "resolve the service descriptions for a LagomScala project" in {
+      val expectedJsonString =
+        """
+          |[
+          |  {
+          |    "name": "/aclservice",
+          |    "acls": [
+          |      {
+          |        "method": "GET",
+          |        "pathPattern": "\\Q/scala-mocks/\\E([^/]+)"
+          |      },
+          |      {
+          |        "method": "POST",
+          |        "pathPattern": "\\Q/scala-mocks\\E"
+          |      }
+          |    ]
+          |  },
+          |  {
+          |    "name": "/noaclservice",
+          |    "acls": []
+          |  }
+          |]
+        """.stripMargin
+
+      val actualJsonString = ServiceDetector.services(this.getClass.getClassLoader, classOf[StubServiceLoader].getName)
```

Comment:
This is the only relevant bit of this PR: introduce a new test for the `scaladsl`.